tozny/resource_account.go

```diff
@@ -22,12 +139,70 @@ func resourceAccount() *schema.Resource {
         ForceNew:    true,
         Elem: &schema.Resource{
```

Comment:
For all of the crypto values that have required fields, can those be generated by the module and then stored as sensitive values, if not provided?

Reply:
yes, but that just omits them from command line output, it still stores them as plain text in the terraform state. And though almost (if not all) of the values currently on display here are public key type crypto values, the recommended use case will be to read and store sensitive Tozny account and client files from a `~/.tozny/e3db.json` file